src/emulator/functionsRuntimeWorker.ts

```diff
@@ -63,7 +63,7 @@ export class RuntimeWorker {
     });
   }
 
-  async execute(frb: FunctionsRuntimeBundle, opts?: InvokeRuntimeOpts) {
+  execute(frb: FunctionsRuntimeBundle, opts?: InvokeRuntimeOpts): Promise<void> {
```

Comment:
Seems like this should just be `void`

Reply:
Also true. Didn't require any code change. 😨